crates/astria-proto/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 #[rustfmt::skip]
 mod proto;
 
-pub use proto::tonic::{
+pub use proto::generated::{
```

Comment:
Nit: fine with the rename, although it feels orthogonal to the rest of the PR

Reply:
It is kinda orthogonal, but related. 

I exclude generated lines from line changes but it wasn't clear that's what was happening. This increases that clarity.